tests/loginInvalidUser.test.js

```diff
@@ -0,0 +1,20 @@
+const { test, expect } = require('@playwright/test');
+const { LoginPage } = require('../pages/loginPage.js');
+import { generateRandomString } from '../utils/random.js';
+
+const randomUser = 'invalidUser' + generateRandomString(4);
+const randomPassword = generateRandomString(10);
+
+test.describe('Redmine login', () => {
+    test('should not login with random user and password', async ({ page }) => {
+        const loginPage = new LoginPage(page);
+        await loginPage.goto('https://www.redmine.org/login');
+        const user = {
+            username: randomUser,
+            password: randomPassword,
+        };
+        await loginPage.loginUser(user);
```

Comment:
@karenn32 According to the TC5 ER - we must verify, that the fields have been filled, how is that covered here? There is no assertion to check the field.